code/letsencrypt.tf

```diff
@@ -0,0 +1,22 @@
+provider "acme" {
+  server_url = "https://acme-v02.api.letsencrypt.org/directory"
+}
+
+module "acme_le" {
```

Comment:
@ElfoLiNk this is really cool, but so far we used a different approach: we run this power shell script https://github.com/pagopa/cert-az-management through azure devops weekly which **creates** and **renews** certificates. Furthermore, we need to register a DNS record before. @AleDore do we need a DNS record for UAT? In which public zone? *.pagopa.it. I guess.

Reply:
@uolter: IMHO we can register `test.cgnonboardingportal.pagopa.it `